package.json

```
@@ -4,6 +4,9 @@
   "files": [
     "build/dist/**"
   ],
+  "bin": {
+    "brimcap": "build/dist/brimcap"
+  },
```

Nit: Keep these alphabetized.
```suggestion
  "bin": {
    "brimcap": "build/dist/brimcap"
  },
  "files": [
    "build/dist/**"
  ],
```